### ISSUE TITLE
Input commands issue

### ISSUE THREAD

**Issue by hboxgames** [alloystorm/dvvr#172]
Hi !

Thanks for the great updates as always.
By the way, I am currently using the older version because the last one had some changes that breaks all the controls.

Since I only use one controler for pretty much everything, now it became impossible to navigate in space. Even in the settings, you deleted the option for the forward/backward.
I don't really see if there is improvements in this change, but I would appreciate rolling back at least so that I can do my own settings !

Also, flagging out that models that were blacking out the screen still do or do it again ... I tried the "workaround" section with the NAN thing etc, nothing worked this time.

Thanks keep up the great stuffs, can't wait for the BVH !

**Comment by alloystorm** [comment]
Hi the input controls are still the same except that when you have UI visible, the UI navigation might takes priority and camera control will not get any input in that case. 

This is controlled by a setting in "Input Settings". You can set "UI Navigation" to "None" in the "Input Settings" to stop this behaviour. Actually that should be the default value in the first place if I remember correctly. Let me know if that solves the issue. 

And this was an experimental thing anyways and in the coming release we have something else to replace it.

**Comment by hboxgames** [comment]
Oh really ?
Cause it was more about not just when UI is visible, but while I use only the right controller for example, it was only possible to "turn and go up/down" but impossible to go forward/back. I'll try again

**Comment by alloystorm** [comment]
> 

I see, previously when you hold the grip button the right hand thumbstick up/down controls forward/back, but this is assigned to tilting by default in 1.2.2, you can change it back in input settings.

**Comment by hboxgames** [comment]
Exactly !
Cool, thanks !!

**Comment by hboxgames** [comment]
> > 
> 
> I see, previously when you hold the grip button the right hand thumbstick up/down controls forward/back, but this is assigned to tilting by default in 1.2.2, you can change it back in input settings.


Hi !
Well actually no, we can't change that in the input settings anymore, and that's why I was asking. the input settings became very limited and lot can't be changed, just have to bear with. now I can't simply move around which is totally frustrating. And I don't really see the good of the tilting function being there anyway ...

Also little thing about the UI, The "settings" of the character opens up as a looong list, I have to scroll down to catch the "full settings" which I actually only use since it works best and have everything in it. Unfortunately that asks more effort AND the "reset all" button is just right there after that. And I already clicked twice on it, forcing myself to re-do all the settings .......
It could help already if that "RESET" button had a confirmation message somehow.

Lastly, I tried out several VAR files, being awed by the fact that you made them compatible .... But well, it didn't contain any BVH inside although it has motion info (since its supposed to be a dance) So I was wondering what the deal was, why it doesnt contain BVH files in it for its motions ....

Thanks for the great stuff as always, and will be back with more tests !
